src/sentry/static/sentry/images/logos/logo-komodor.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="680" height="483" viewBox="0 0 680 483" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Have we run this by @doralchan? She's helped me in the past making sure our SVGs are up to code.

Reply:
👍  using the formatted SVG now